### DISCUSSION TITLE
Forking to this organisation

### DISCUSSION THREAD

**Discussion by bobturneruk** [AFM-SPM/home#4] [General]
Should we encourage forking to this organisation? I'd prefer to link to others code as forks can lead to it being unclear where this definitive version can be found.

**Comment by paulmueller** [comment]
I think we should fork code here, because

- It adds to the relevance of this organization
- Forks are redundant copies; I often experienced that other sites go down over the years.

There will be also be other code that is not on any git server (e.g. just source tars) which will get their own repositories in this organization. It is easier to keep track if everything is just here.

I volunteer to set up a separate repository for keeping track of code with issue templates that will allow anyone to request a fork / inclusion of their code.

**Comment by bobturneruk** [comment]
Hi! Both your points make sense to me, but I worry about keeping forks up to date with their upstreams. I think we risk having outdated versions here unless someone is going to actively keep things up to date? Maybe there's some GitHub cleverness that can be employed?

And whilst it's great that you're volunteering to do some of this sort of thing, I think we need to carefully consider where we put our currently limited resources. I don't want you to be overloaded and I would like this organisation to become resilient to people volunteering varying levels of effort.

Perhaps a list of software would (e.g. as a csv or markdown in the home repo.), for the the time being, make most sense?

**Comment by paulmueller** [comment]
I consider this a one-time job, so I think that should be fine. Repositories can be kept up-to-date via monthly cron jobs using CI services. It's also not that important to have up-to-date forks as long as the original repository is still around.

**Comment by bobturneruk** [comment]
OK, then! I guess we just need to make sure that folks not familiar with GitHub and it's forks and branches can easily find what they need - I think we've all made statements to this effect already!
